quikey/quikey.py

```diff
@@ -92,6 +95,27 @@ def ls(ctx, show_all):
     output = AsciiTable(table)
     click.echo(output.table)
 
+@cli.command()
+@click.option('--location', '-l', default=BaseDirectory.xdg_config_home+"/autokey/data/", show_default=True, help='Location of top level directory to import from autokey')
+@click.pass_context
+def keyimport(ctx,location):
+    tags = ['autokey-imports']
+    db = ctx.obj['database']
+    importfiles = PhraseFind(location)
+    for key in importfiles:
+        contents = None
+        if importfiles[key] is not None:  
+            contents = importfiles[key]
+        else:
+            contents = click.edit('\n\n'+MARKER)
+            if contents is not None:
+                contents = contents.split(MARKER, 1)[0].rstrip('\n')
+            else:
+                click.echo('quikey phrase with key of %s not added' % key)
+                return
+        db.put(key, contents, tags)
```

Comment:
Can you check if the phrase key exists already before inserting it? I found this to be a problem if I can the key import multiple times. 

Note, this is a problem in the `add` command too and I've filed an issue here: #43

Reply:
I was thinking about that and then I forgot to do anything about it. If you make the check for add easily accessible as a function we can likely use the same check for both add and import.

Let me know what you'd like to do before we both go writing checks for the same thing.

Reply:
Added checks on keyimport that match the way #43 was resolved.